helm/aio-connector-multidataset/templates/aio-connector-asset.yaml

```diff
@@ -1,19 +1,12 @@
 ---
-apiVersion: deviceregistry.microsoft.com/v1
+apiVersion: namespaces.deviceregistry.microsoft.com/v1beta1
 kind: Asset
 metadata:
-  name: {{ .Values.asset.name | default "device-001" }}
+  name: {{ .Values.device.name | default "device-001" }}
```

Comment:
Wonder if this is always going to be the case i.e. assetname == device name? or it is just a typo?

Reply:
I was a bit confused as we use the same name for kinds `Asset` and `Device`. Changed back to `assetName` now.